Sources/Brave/Assets/About/Licenses.html

```diff
@@ -686,39 +686,6 @@ <h4 id="exhibit-b---incompatible-with-secondary-licenses-notice">Exhibit B - “
 
 <p class="center">-</p>
 
-<div>
```

Comment:
Can you also update line 846 where Feedkit is written ZIP foundation

```
    <input id="ac-FeedKit" name="accordion-1" type="checkbox" />
    <label for="ac-FeedKit"><h2>ZIP Foundation</h2></label>
```

    <input id="ac-FeedKit" name="accordion-1" type="checkbox" />
    <label for="ac-FeedKit"><h2>FeedKit</h2></label>

Reply:
Made the change thanks. And removed `pop` library from licenses